gems/pending/disk/modules/VhdxDiskProbe.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: US-ASCII
+
+
+VHDX_DISK      = "VhdxDisk"
+VHDX_SIGNATURE = "vhdxfile"
+
+module VhdxDiskProbe
+  def VhdxDiskProbe.probe(ostruct)
```

Comment:
Because when I wrote the code back in March most if not all of the other Probe files used the class.probe convention.  I will change it to be consistent with the current convention.